packages/fx-core/src/core/environment.ts

```diff
@@ -385,6 +392,50 @@ class EnvironmentManager {
       return this.defaultEnvName;
     }
   }
+
+  public getActiveEnv(projectRoot: string): Result<string, FxError> {
+    if (!isMultiEnvEnabled()) {
+      return ok("default");
+    }
+    if (!isValidProject(projectRoot)) {
```

Comment:
nit: keep return consistent error in envManager if project root path doesn't exist.
Other APIs return `PathNotExistError` while `getActiveEnv` return `InvalidProjectError`:
```ts
if (!(await fs.pathExists(projectPath))) {
  return err(PathNotExistError(projectPath));
}
```

Reply:
Changed `PathNotExistsError` to be consistent with other APIs.